packages/dev/docs/pages/contribute.mdx

```diff
@@ -16,135 +17,4 @@ description: How to contribute features, report bugs, or make suggestions to imp
-cd react-spectrum
-yarn install
-```
-
-You can then run the storybook and browse to [http://localhost:9003](http://localhost:9003) with:
-```bash
-yarn start
-```
-
-Or run the documentation and browse to [http://localhost:1234/](http://localhost:1234/) with:
-```bash
-yarn start:docs
-```
-
-### Tests
-We use [jest](https://jestjs.io/) for unit tests and [react-testing-library](https://testing-library.com/docs/react-testing-library/intro) for rendering and writing assertions. Please make sure you include tests with your pull requests. Our CI will run the tests on PRs as well as the linter and type checker. You can see on each PR whether you have passed all our checks. We split the tests into 2 groups.
-
-*Visual tests*
-- A Storybook story should be written for each visual state that a component can be in (based on props).
-
-*Unit tests*
-- (Props) Anything that should be changed by a prop should be tested via react-testing-library.
-- (Events) Anything that should trigger an event should be tested via react-testing-library.
-
-You can run the tests with:
-
-```bash
-yarn jest
-```
-
-You can also get a code coverage report by running:
-
-```bash
-yarn jest --coverage
-```
-
-### Linting
-The code is linted with [eslint](https://eslint.org/). The linter runs whenever you run the tests, but you can also run it with
-```bash
-yarn lint
-```
-
-### TypeScript
-The code for React Spectrum is written in [TypeScript](https://www.typescriptlang.org/). The type checker will usually run in your editor, but also runs when you run
-```bash
-yarn lint
-```
-
-### Storybook
-We use [Storybook](https://storybooks.js.org) for local development. Run the following command to start it:
-```bash
-yarn start
-```
-Then, open [http://localhost:9003](http://localhost:9003) in your browser to play around with the components and test your changes.
-
-### Documentation
-Our documentation should always remain up to date. When making changes to components, make sure the appropriate documentation has been updated to reflect those changes. Documentation for each component can be found in the docs folder within a component's package. Other documentation pages can be found in the [packages/dev/docs](https://github.com/adobe/react-spectrum/tree/main/packages/dev/docs) folder in the codebase.
-
-Documentation can be run locally by using
-```bash
-yarn start:docs
-```
-Then, open [http://localhost:1234](http://localhost:1234) in your browser.
-
-## Contributor to committer
-
-We love contributions from our community! If you'd like to go a step beyond contributor and become a committer with full write access and a say in the project, you must be invited. The existing committers employ an internal nomination process that must reach lazy consensus (silence is approval) before invitations are issued. If you feel you are qualified and want to get more deeply involved, feel free to reach out to existing committers to have a conversation.
+<Contributing />
```

Comment:
if you place that before mentioned `# Contributing` right above this line, the side nav will have the link text. It's not looking inside the rendered `<Contributing` component

Reply:
Oh.
So it does not evaluate the mdx document before it does the frontmatter stuff?

Guess # Contributing would be duplicated then?

Reply:
yeah, that duplicates it unfortunately

Reply:
Bummer.

I can either remove the commit which does ghe mdx import.
Or the tool which generates these links needs a change.

Reply:
@snowystinger 
I added the `title` frontmatter. Now it should work